src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotEntries = Object.entries(robot);
```

Comment:
You have a working solution, but let me ask you: do you really need to store in the memory one array of robot entries?

Maybe could check the key in the newly created object?

Reply:
I agree, no need to create a new array, fixed